### DISCUSSION TITLE
Length mismatch: Expected axis has 4 elements, new values have 5 elements | team name = cciip

### DISCUSSION THREAD

**Discussion by LZY-the-boys** [PaddlePaddle/PaddleSpatial#153] [BaiduKDDCup2022] [answered]
![image](https://user-images.githubusercontent.com/72137647/172275497-26516fdb-4a18-4e85-8bba-55df97c30901.png)
My submission had this problem, which didn't occur when I run at local yesterday with uptodate evaluation code.
Could you please show me what is wrong or show the full traceback stack?

**Comment by ramber1836** [comment, marked as answer]
<img width="1243" alt="image" src="https://user-images.githubusercontent.com/19561583/172279343-1783ebf6-7b88-488f-ab9d-eff485c3fb1a.png">
